config.yml

```diff
@@ -3,6 +3,7 @@ defaults:
   app:
     jobQueue: null
     statusQueue: null
```

Comment:
This queue will later be used for both Statuses and Checks. The former tracks group status, and the latter - well, it will depend. I was thinking of `resultStatus` <--- what do you think of that?